app/models/tag.rb

```diff
@@ -222,4 +222,11 @@ def self.trending(limit = 5 , start_date = DateTime.now - 1.month , end_date = D
        .order('count DESC')
        .limit(limit)
   end
+
+  #select nodes by tagname and user_id
+ def self.tagged_nodes_by_author(tagname, user_id)
+    Node.where('term_data.name = ?', tagname)
+       .includes(:node_tag, :tag)
```

Comment:
OK, in Rails 4, we also have to say `.references(:term_data)` if we're going to use that in a string-based `where()` filter. I'm getting used to this myself!

Reply:
I'm not 100% sure of this, so maybe make the other change first and we'll see how this does? If it still fails, then this change may also be needed. Also, if you could get all the `.` periods to align, it really does help keep the codebase tidy, and I'd appreciate it! Thanks!